clang_build/target.py

```diff
@@ -227,21 +227,8 @@ def __init__(self,
         # Linking setup
         self.linkCommand = link_command + [str(self.outfile)]
 
-        ### Library dependency search paths
-        for target in self.dependencyTargets:
```

Comment:
Why is this removed and *duplicated* in subclasses?

Reply:
Only to keep the ordering of command line parameters, flags etc. in the command consistent between different targets, i.e. first comes the outfile, then the own flags, then potentially library search directories, ...